src/services/AccountService/galoy-account-service.ts

```diff
@@ -1,46 +1,57 @@
 import { GALOY_GRAPHQL_ENDPOINT } from "../../config/index"
 import fetch from "node-fetch"
 
-const GALOY_GRAPHQL_ENDPOINT_QUERY = `query{
-  allWallets(walletCurrency:BTC)
-  {
+const getAccountIds = async () => {
+  const query = `query{
+    listWalletIds(walletCurrency:BTC)
+  }`
+  const response = await fetch(GALOY_GRAPHQL_ENDPOINT, {
+    method: "POST",
+    headers: {
+      "Content-Type": "application/json",
+      "Accept": "application/json",
+    },
+    body: JSON.stringify({ query }),
+  })
+  const data = await response.json()
+  return data.data.listWalletIds
+}
+
+const queryBuilderToFetchWallet = (id: string): string => {
+  return `query {
```

Comment:
query should be named 

ie:
```suggestion
  return `query MyQuery {
```